modules/Vtiger/duplicate.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+error_reporting("E_ALL & ~E_NOTICE & ~E_STRICT & ~E_DEPRECATED & ~E_WARNING");
+require_once "include/utils/utils.php";
+
+if(isset($_REQUEST['module_name']) && isset($_REQUEST['record_id']))
+{
+	$currentModule = vtlib_purify($_REQUEST['module_name']);
+	$record_id = vtlib_purify($_REQUEST['record_id']);
+	
+	require_once 'modules/'.$currentModule.'/'.$currentModule.'.php';
+	$focus = new $currentModule();
+	$focus->retrieve_entity_info($record_id, $currentModule);
+
+	//Duplicate Records that this Record is dependent of 
```

Comment:
We don't need this block. Our specification is to duplicate dependent and related modules, not directly related records. Please comment this piece of code as it may be useful in the future.